man/bootctl.xml

```diff
@@ -237,6 +237,14 @@
         cannot be written. Currently only applies to random seed operations.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--make-machine-id-directory[=yes|no|auto]</option></term>
+        <listitem><para>Control creation and deletion of <literal>$ESP/$MACHINE_ID</literal> during
+        <option>install</option> and <option>remove</option>, respectively.
+        <literal>auto</literal> decays to <literal>yes</literal> if <literal>/etc/machine-id</literal>
```

Comment:
"decays" sounds a bit negative. Maybe say "is equivalent to … and otherwise to"

Reply:
hasn't occurred to me that someone might have a different association than array decay lol

Updated.